app/controllers/complaints_controller.rb

```diff
@@ -0,0 +1,3 @@
+# frozen_string_literal: true
+class ComplaintsController < ApplicationController
+end
```

Comment:
@mphprogrammer why are these stubbed controllers necessary for this PR? Can they wait until these are actually implemented in a separate PR?